### PR TITLE
[DPE-3724] Initial commit

### PR DESCRIPTION
**Pull request by welpaolo**

### CODE REVIEW

File: files/spark/conf/spark-defaults.conf [hunk 1]
```diff
@@ -0,0 +1 @@
+spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04
```
Comment (deusebio) [inline files/spark/conf/spark-defaults.conf]: Shouldn't this be 

```suggestion
spark.kubernetes.container.image=ghcr.io/canonical/charmed-spark:3.4-22.04_{risk}
```

using either edge or stable?

**Review by deusebio — APPROVED** [review]
Looks good! Just a small possibly typo

File: files/spark/conf/spark-defaults.conf (hunk 1 above)
Comment (welpaolo) [reply files/spark/conf/spark-defaults.conf]: done

**Review by theoctober19th — APPROVED** [review]
LGTM

**Pull request merged**